app/helpers/generic_file_helper.rb

```diff
@@ -39,7 +39,19 @@ def download_path(*args)
       gf = GenericFile.find(item.id)
       return gf.doi_url if gf.doi_url.present?
     end
-    return sufia.download_path(*args)
+    if args[1].nil?
+      return "/files/" + item.id + "/" + URI::encode(item.label) unless item.label.nil?
+      return "/files/" + item.id + "/nofile"
```

Comment:
Sorry, "empty element" is kind of vague. What I mean is, rather than render a link that will intentionally 404, we should probably just not render a download link at all. Google tends to punish pages with dead links on them, and it would make more sense to remove the download UI element entirely in cases where it doesn't apply.